next/src/components/pdf/ChatWithPdf.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { motion } from "framer-motion";
+import { AiOutlineArrowUp } from "react-icons/ai";
+import { streamTextWithoutAgentBody } from "../../services/stream-utils";
+import MarkdownRenderer from '../../components/console/MarkdownRenderer';
+import { useAuth } from '../../hooks/useAuth';
+import { useRouter } from "next/router";
+
+interface Message {
+    id: number;
+    role: string;
+    content: string;
+}
+
+const ChatWithPdfComponent: React.FC = () => {
```

Comment:
```suggestion
const ChatWithPdfComponent: FC = () => {
```

Nit we can just import the FC type instead of all of react